README.md

```diff
@@ -12,3 +12,7 @@ features such as optional chaining, nullish coalescing,
 and private `class` fields/methods; please see the `legacy/` folder.
 
 See https://github.com/mozilla/pdf.js for learning and contributing.
+
+# Releases
+
+As of June 2024, latest releases are pushed directly to **[NPM](https://www.npmjs.com/package/pdfjs-dist)** and may not be visible here.
```

Comment:
The problem started a lot earlier, so we should probably provide a more accurate date.
```suggestion
As of February 2024, releases are pushed directly to **[NPM](https://www.npmjs.com/package/pdfjs-dist)** and may not be visible here.
```